### DISCUSSION TITLE
Lodestar Standup 2024-9 (March 5, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6493] [Standups]
### Meeting Info
[Mar 5, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/mar-5-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v1.16.1 optimizations
  - Monthly PR Scrub
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
Would like to bring up with @jeluard about an improvement to docs via Docusaurus + mdx features to align with something nice that also incentivizes more contributions to docs. See https://docs.buildwithsygma.com/ . Would like to see the appetite for migrating here as long as it doesn’t increase any burden for automating things.

**Reply by nflaig** [reply]
> Would like to see the appetite for migrating here as long as it doesn’t increase any burden for automating things.

Would be great to give a summary here before the meeting, and answer some questions beforehand
- what problem does it address in our docs (besides the fact that we lack content)
- what are the main features we wanna utilize that our current docs tooling is lacking
- how does it incentivizes more contributions to docs

**Reply by jeluard** [reply]
Thanks @philknows for initiating this!

Some areas I think could be improved
* current docs looks like it's [1999](https://www.youtube.com/watch?v=rblt2EtFfC4), and not in a good way :) I believe a nice env would foster team appetite to write docs; it would definitely be more appealing for end users
* flat hierarchy doesn't allow to easily navigate content
* supporting mdx (on top of markdown) would allow to show off lodestar libs (e.g. `prover` or `light-client`) directly in the web page
* better api access would simplify build time content generation (as opposed to current [manual](https://github.com/ChainSafe/lodestar/blob/unstable/packages/cli/docsgen/index.ts) [generation](https://github.com/ChainSafe/lodestar/blob/unstable/scripts/prepare-docs.sh))
* with more advanced framework like [astro](https://astro.build/) both [main page](https://lodestar.chainsafe.io/), docs and blog could form a unified website, akin to https://buildwithsygma.com/

**Reply by jeluard** [reply]
Also relevant: https://github.com/ChainSafe/lodestar/issues/5961

**Reply by matthewkeil** [reply]
I tend to agree that a react based markdown framework will better align with our process.  feels a bit funny to use python in a TS library for the docs.  will also open us up to custom react components should we want a couple here or there.   also will be nice to not have page reloads when the section changes.

I have used Docusaurus and a couple others (Gatsby, etc) and all are about the same in my opinion.  Its been a while so guessing there are some new frameworks on the horizon.  Thanks @jeluard for doing some research to figure out the right one to use

**Comment by philknows** [comment]
Tomorrow we will also be going over a decision to pre-release v1.16.1-rc.0 for testing with @g11tech and the status of #6499. Please bring up any other optimizations that need to be put into testing. This release will be an optimization update for performance but not mandatory for the Deneb fork.

**Comment by philknows** [comment]
In addition, tomorrow is monthly PR grooming day. Please resolve or come prepared with your open PRs for updates/discussion.

**Comment by philknows** [comment]
Closing for #6510 . 
Notes for this session are located here: https://github.com/ChainSafe/lodestar/wiki/Lodestar-Planning-&-Standup-Meetings#march-5-2024-planning-and-standup-meeting
